assets/styles/hydrogen/_variables.scss

```
@@ -0,0 +1,3 @@
+$family-primary: 'Gidole', sans-serif;
+$family-titles: Blogger Sans, sans-serif;
+$box-shawdow-coac:  0 8px 8px 0 rgba(0,0,0,.05);
```

It's `shadow` instead of `shawdow`
```suggestion
$box-shadow-coac:  0 8px 8px 0 rgba(0,0,0,.05);
```